cmd/gateway/commands.go

```diff
@@ -100,11 +138,18 @@ func createStaticModeCommand() *cobra.Command {
 				return fmt.Errorf("error validating POD_IP environment variable: %w", err)
 			}
 
+			var gwNsName *types.NamespacedName
+			if !gateway.Empty() {
```

Comment:
Can you use this [function](https://github.com/spf13/pflag/blob/d5e0c0615acee7028e1e2740a11102313be88de1/flag.go#L515) on the gateway flag instead?